src/SIS_slow_thermo.F90

```diff
@@ -59,6 +59,8 @@ module SIS_slow_thermo
 use SIS2_ice_thm,      only : ice_resize_SIS2, add_frazil_SIS2, rebalance_ice_layers
 use SIS2_ice_thm,      only : get_SIS2_thermo_coefs, enthalpy_liquid_freeze
 use SIS2_ice_thm,      only : enth_from_TS, Temp_from_En_S, enthalpy_liquid, calculate_T_freeze
+use SIS_sponge,        only : initialize_icerelax_file, set_up_isponge_field, apply_isponge, isponge_CS
+use SIS_sponge,        only : global_to_local_ij
 
```

Comment:
`global_to_local_ij()` and `set_up_isponge_feild` are not used in SIS_slow_thermo.F90, and hence should be remove from these `use ..., only` statements.